flink-table-store-core/src/main/java/org/apache/flink/table/store/file/mergetree/SortBufferMemTable.java

```diff
@@ -124,7 +124,7 @@ void assertBufferEmpty() {
     private class MergeIterator implements Iterator<KeyValue> {
         private final MutableObjectIterator<BinaryRowData> kvIter;
         private final Comparator<RowData> keyComparator;
-        private final MergeFunctionHelper mergeFunctionHelper;
+        private final MergeFunction<KeyValue> mergeFunction;
```

Comment:
I'm a little hesitant to change it to MergeFunction, so it doesn't know whether the MergeFunction has been wrapped or where it is appropriate to wrap it.